### DISCUSSION TITLE
border-b does not work inside of divide-y

### DISCUSSION THREAD

**Discussion by DaviMenezes** [tailwindlabs/tailwindcss#8371] [Help] [answered]
![image](https://user-images.githubusercontent.com/3998868/168909791-686291e5-8f46-42a7-9b89-38b7915a546c.png)
![image](https://user-images.githubusercontent.com/3998868/168909989-f8b66941-a290-47d0-87af-71f433cc42e4.png)

**Comment by lh536** [comment]
![image](https://user-images.githubusercontent.com/37967244/168917091-8213f5d8-0431-4f6c-b177-4d7a32edf044.png)

Maybe it's not the best solution... but at least it works.

**Reply by DaviMenezes** [reply]
the responsibility of the upper and lower borders in this case needs to be of the internal elements, but thanks for help

**Comment by DaviMenezes** [comment, marked as answer]
the solution via modifier "!important" (native css) usaded only as "!", so, `<div class="!border-b">` works
https://tailwindcss.com/docs/configuration#important-modifier
